bower.json

```diff
@@ -10,7 +10,8 @@
   "component" : "hotgraphic",
   "assetFields" : [
     "_graphic.src",
-    "_items[]._graphic.src"
+    "_items[]._graphic.src",
```

Comment:
you don't need to update this, it hasn't been needed/used in ages. actually being deleted in #141